app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::UsersController < ApplicationController
+  def show
+    render json: User.find(params[:id])
+  end
+end
```

Comment:
Lets add a controller test to make sure this is returning the correct json object that we are expecting